plonky3/src/prover/params.rs

```diff
@@ -0,0 +1,98 @@
+//! The concrete parameters used in the prover
+//! Inspired from [this example](https://github.com/Plonky3/Plonky3/blob/6a1b0710fdf85136d0fdd645b92933615867740a/keccak-air/examples/prove_goldilocks_keccak.rs#L57)
```

Comment:
OK, this looks like something a cryptographer should have a say on. I guess for now it doesn't really matter, except if it has performance implications?

Reply:
Agreed